ddtrace/appsec/iast/processor.py

```diff
@@ -45,7 +49,7 @@ def on_span_finish(self, span):
         data = _context.get_item(IAST.CONTEXT_KEY, span=span)
 
         if data:
-            span.set_tag_str(IAST.JSON, json.dumps(attr.asdict(data)))
+            span.set_tag_str(IAST.JSON, json.dumps(attr.asdict(data, filter=filter_none)))
```

Comment:
Instead of  `filter_none` what do you think about:

```suggestion
            span.set_tag_str(IAST.JSON, json.dumps(attr.asdict(data, filter=lambda x: x is not None)))
```